Csharp/sigmf/sigmf/MetaDoc.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.IO;
+using Newtonsoft.Json;
+
+namespace sigmf
+{
+    [Serializable()]
+    public class MetaDoc
+    {
+        private static readonly long serialVersionUID = 1L;
+
+        protected string id;
+
+        protected Globals global;
+
+        protected List<Capture> captures;
```

Comment:
Since this is the root of the metadata, I believe you need to tag these properties with JSON attributes.